phantom-dsl/src/main/scala/com/websudos/phantom/CassandraTable.scala

```diff
@@ -205,7 +210,13 @@ abstract class CassandraTable[T <: CassandraTable[T, R], R] extends SelectTable[
 
     columnMembers.foreach {
       symbol =>
-        val column = instanceMirror.reflectModule(symbol.asModule).instance
+
+        val column = if (symbol.isModule) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/flavian/phantom_2/file/2212474154/issues/source?bid=2847375&fileBranchId=3164226#l214)